File: CRM/src/app/shared/eHospital_DongNai_A/Sync_KeyOffline.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sync_KeyOffline } from './Sync_KeyOffline.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sync_KeyOfflineService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'IsUpdate', 'TableName', 'UpdatedDate'];

    List: Sync_KeyOffline[] | undefined;
    ListFilter: Sync_KeyOffline[] | undefined;
    FormData!: Sync_KeyOffline;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sync_KeyOffline";
    }
}

File: Service_eHospital_DongNai_A/Implement/Sync_KeyOfflineService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Sync_KeyOfflineService : BaseService<Sync_KeyOffline, ISync_KeyOfflineRepository>
    , ISync_KeyOfflineService
    {
    private readonly ISync_KeyOfflineRepository _Sync_KeyOfflineRepository;
    public Sync_KeyOfflineService(ISync_KeyOfflineRepository Sync_KeyOfflineRepository) : base(Sync_KeyOfflineRepository)
    {
    _Sync_KeyOfflineRepository = Sync_KeyOfflineRepository;
    }
    }
    }

